chroma-manager/tests/framework/unit/jenkins_steps/main

```diff
@@ -6,13 +6,26 @@
 set_defaults false
 check_for_autopass
 
+# figure out which parallel invocation we might be
+WORKSPACE_NUMBER=${WORKSPACE/$HOME\/workspace\/unit-tests@/}
+WORKSPACE_NUMBER=${WORKSPACE_NUMBER%%/*}
+
+if [ -n "$WORKSPACE_NUMBER" ]; then
+    # use the database created for the workspace number
+    ed <<EOF $CHROMA_DIR/chroma-manager/settings.py
+/'NAME': 'chroma'/s/',/$WORKSPACE_NUMBER',/
```

Comment:
Sure.  And as noted, it's pretty rigid about the database name.  I will follow-up with a patch to use `local_settings.py` and be flexible to any database name.